### PR TITLE
[7.67.x][BXMSPROD-2057] Updating references to the new kie-ci actions folder

### PR DESCRIPTION
**Pull request by bacciotti**
Related PR's:
- https://github.com/kiegroup/optaplanner/pull/2806
- https://github.com/kiegroup/optaplanner/pull/2837

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -27,18 +27,19 @@ jobs:
...
+        uses: kiegroup/kie-ci/.ci/actions/maven@main
         with:
           java-version: ${{ matrix.java-version }}
           maven-version: ${{ matrix.maven-version }}
           cache-key-prefix: ${{ runner.os }}-${{ matrix.java-version }}-maven${{ matrix.maven-version }}
       - name: Build Chain
-        uses: kiegroup/droolsjbpm-build-bootstrap/.ci/actions/build-chain@main
+        uses: kiegroup/kie-ci/.ci/actions/build-chain@main
         with:
+          definition-file: https://raw.githubusercontent.com/${GROUP:kiegroup}/kogito-pipelines/${BRANCH:main}/.ci/pull-request-config.yaml
```
Comment (Ginxo) [inline .github/workflows/pull_request.yml:40]: I would say this should point to droolsjbpm-build-bootstrap one since 7.67.x belongs to RHPAM

**Review by Ginxo — CHANGES_REQUESTED** [review]
please check

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (lampajr) [reply .github/workflows/pull_request.yml:40]: I agree @Ginxo

**Review by Ginxo — APPROVED** [review]
LGTM

**Pull request merged**